dipy/direction/peaks.py

```diff
@@ -552,15 +551,6 @@ def peaks_from_model(model, data, sphere, relative_peak_threshold,
                            odf_array if return_odf else None)
 
 
-def gfa(samples):
-    """The general fractional anisotropy of a function evaluated
```

Comment:
This would not affect users (and I believe that it wouldn't even affect the aformentioned tests!), because we are now importing the name `gfa` in from `reconst.odf` into `direction.peaks`. So the API hasn't changed at all.  

The only ones who might be surprised are developers who expect to find this function here. What do you propose to do to warn them about this?